src/Blueprint/Blueprint/help/Get-AzBlueprint.md

```diff
@@ -0,0 +1,214 @@
+```
+
+Get the lastest published blueprint definition with the given name within the specified management group.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -LatestPublished
+The latest published blueprint definition flag.
+When set, execution returns the latest published version of the blueprint definition.
+Defaults to false.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: BySubscriptionNameAndLatestPublished, ByManagementGroupNameAndLatestPublished
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ManagementGroupId
+Management Group Id where the blueprint definition is saved.
+
+```yaml
+Type: System.String
+Parameter Sets: ManagementGroupScope, ByManagementGroupAndName, ByManagementGroupNameAndVersion, ByManagementGroupNameAndLatestPublished
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Name
+Blueprint definition name.
+
+```yaml
+Type: System.String
+Parameter Sets: BySubscriptionAndName, BySubscriptionNameAndVersion, BySubscriptionNameAndLatestPublished, ByManagementGroupAndName, ByManagementGroupNameAndVersion, ByManagementGroupNameAndLatestPublished
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -SubscriptionId
+Subscription Id where the blueprint definition is saved.
+
+```yaml
+Type: System.String
+Parameter Sets: SubscriptionScope, BySubscriptionAndName, BySubscriptionNameAndVersion, BySubscriptionNameAndLatestPublished
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Version
+Published blueprint definition version.
+
+```yaml
+Type: System.String
+Parameter Sets: BySubscriptionNameAndVersion, ByManagementGroupNameAndVersion
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### System.String
+
+## OUTPUTS
+
+### System.Object
```

Comment:
@filizt you'll need to regenerate the help for these cmdlets again once you've finalized all your changes since some of this information will be outdated

Reply:
Ok - keeping this feedback active to track at the very end.

---
In reply to: [261405912](https://github.com/Azure/azure-powershell/pull/8537#discussion_r261405912) [](ancestors = 261405912)